### ISSUE TITLE
Button's width, height and background can not be set or changed. And one more question, how to put or post messages?

### ISSUE THREAD

**Issue by Tumiz** [qmlweb/qmlweb#413]
Button's width, height or background can not be set or changed. It is always like a small button with a grey  background. 
Another question is how to put or post messages? Please give me some examples.

**Comment by ChALkeR** [comment]
> Button's width, height or background can not be set or changed. It is always like a small button with a grey background.

Yes, that is indeed a bug — Button is not styleable yet. That needs to be fixed.
The file in question is https://github.com/qmlweb/qmlweb/blob/master/src/modules/QtQuick.Controls/Button.js

---

> Another question is how to put or post messages? Please give me some examples.

Please don't group several reports/questions in one issue.

PUT/POST could be achieved using XMLHttpRequest, see http://doc.qt.io/qt-5/qtqml-javascript-qmlglobalobject.html#xmlhttprequest and http://doc.qt.io/qt-5/qtqml-xmlhttprequest-methods-js.html for that.

To my knowledge, that is supported in QmlWeb, if you find problems using that — please file individual issues with code testcases to reproduce.

---

I am renaming this issue to track the Button properties.
